src/components/SearchList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useState } from 'react';
+import { normalizeValue } from './Helper';
+
+export default function SearchList(listItems) {
+  const [searchedItem, setSearchedItem] = useState('');
+  const searchInputRef = useRef();
+  //function to update list of items as user enters characters
+  const updateSearch = () => {
+    setSearchedItem(searchInputRef.current.value);
+  };
+
+  const clearSearch = () => {
+    setSearchedItem('');
+    searchInputRef.current.value = '';
+  };
+
+  //function to return list of items whose normalized names contain characters entered by the user
+
+  const filteredList = listItems.list.filter((item) => {
+    const normalizedSearch = normalizeValue(searchedItem);
+    return item.itemNameNormalize.includes(normalizedSearch);
+  });
+
+  return (
+    <div>
+      Filter Items:
+      <input
+        type="text"
+        ref={searchInputRef}
+        placeholder="Search for Items here"
+        onChange={updateSearch}
+      />
+      {searchedItem && <button onClick={clearSearch}>X</button>}
+      <div id="filtered-items">
+        {filteredList.map((item) => {
+          return (
+            <div key={item.id}>
+              <ul>{item.itemName}</ul>
```

Comment:
Accessibility note:  This mapping should render `li` items within a `ul` rather than create a new list for each item. While from a sighted user the difference is not significant for a user utilizing a screen reader it has a big impact on not only the experience but also the understanding of where they are on the page. 

| current  | as a list |
| ------------- | ------------- |
| <video width-400px src="https://user-images.githubusercontent.com/19564687/140625752-1f66dbd5-0a98-4fc1-a740-e2d4e0079e63.mov" />  | <video width-400px src="https://user-images.githubusercontent.com/19564687/140625772-5b311934-b017-40cc-a1f3-02018cfb20b7.mov" />  |

Reply:
Thanks for the feedback and videos. I have made the changes accordingly!